src/components/Comments/ConnectorWrapper.js

```diff
@@ -15,8 +15,8 @@ export const styles = StyleSheet.create({
     display: 'flex',
     flexDirection: 'row'
   },
-  connector: (index) => ({
-    marginLeft: -1,
+  connector: (index, level) => ({
+    marginLeft: level === 2 ? 0 : -1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional logic for `marginLeft` based on `level` could potentially lead to misalignment if `level` is not strictly 1 or 2. Consider adding a default case to handle other potential values of `level`.
```diff
- marginLeft: level === 2 ? 0 : -1,
+ marginLeft: level === 2 ? 0 : (level === 1 ? -1 : /*default value*/),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->